### PR TITLE
Feat: customize the login form using an environment variable

### PR DESCRIPTION
**Pull request by lalver1**
This PR sets the login form configuration using an environment variable.
Before merging this PR, ensure that:
- [ ] `dev` has a new secret in the key vault and a new environment variable that refers to this secret in the app settings
- [ ] `test` has a new secret in the key vault and a new environment variable that refers to this secret in the app settings
- [ ] `prod` has a new secret in the key vault and a new environment variable that refers to this secret in the app settings

### CODE REVIEW

File: benefits/settings.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def RUNTIME_ENVIRONMENT():
     "https://www.googleapis.com/auth/userinfo.email",
     "https://www.googleapis.com/auth/userinfo.profile",
 ]
-SSO_SHOW_FORM_ON_ADMIN_PAGE = False
+SSO_SHOW_FORM_ON_ADMIN_PAGE = os.environ.get("SSO_SHOW_FORM_ON_ADMIN_PAGE", "False").lower() == "true"
```
Comment (machikoyasuda) [inline benefits/settings.py:75]: https://megalus.github.io/django-google-sso/customize/#hiding-the-login-form

**Review by machikoyasuda — APPROVED** [review]
Tested:
image

**Review by angela-tran — APPROVED** [review]
This LGTM!

**Review by angela-tran — CHANGES_REQUESTED** [review]
Luis noticed that there is a missing comma in the line above the one he added 😅 so I'm dismissing my review

**Review by angela-tran — APPROVED** [review]

**Comment by lalver1** [comment]
It's interesting @angela-tran , looks like commas are not required according to the [terraform docs](https://developer.hashicorp.com/terraform/language/expressions/types#maps-objects) since [app_settings](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service#app_settings) are a key-value pair, but it's prob best to include them (it also makes the VS Code syntax highlighting look uniform).

**Pull request merged**